Source/WebInspectorUI/UserInterface/Protocol/Target.js

```diff
@@ -98,6 +98,8 @@ WI.Target = class Target extends WI.Object
             // our breakpoints were already sent to that backend.
             if (this.hasDomain("Inspector"))
                 this.InspectorAgent.initialized();
+            else if (this.hasDomain("ServiceWorker"))
+                this.ServiceWorkerAgent.initialized();
```

Comment:
the proper compatibility check here would be `this.hasCommand("ServiceWorker.initialized")` since the `ServiceWorker` domain has existed for a while whereas the `initialized` command is new

though see my above comment about how we could just expose the `Inspector` domain instead of duplicating this logic elsewhere